run/logging-manual/index.js

```diff
@@ -0,0 +1,26 @@
+// Copyright 2019 Google LLC. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+const app = require('./app');
+const metadata = require('./metadata');
+const PORT = process.env.PORT || 8080;
+const SERVICE = require('./package').name;
+
+const startServer = () => {
+  app.listen(PORT, () => console.log(`${SERVICE} listening on port ${PORT}`));
+}
+
+if (!process.env.GOOGLE_CLOUD_PROJECT) {
+  metadata.getProjectId()
+    .then(project => {
```

Comment:
Use [`await` instead of `then`](https://github.com/xjamundx/eslint-plugin-promise/blob/master/docs/rules/prefer-await-to-then.md).

(If you run `eslint` from the repo root, this should show up as a `WARNING`. **Please fix those `WARNING`s where possible**, as they will eventually be converted to `ERROR`s.)